airflow/api_connexion/openapi/v1.yaml

```diff
@@ -3921,8 +3934,15 @@ components:
         - type: object
           properties:
             value:
-              type: string
-              description: The value
+              anyOf:
```

Comment:
This change for me produces a java code that is not compiled anymore with `2.10.5`.
```
  private AnyOfstringnumberintegerbooleanarrayobject value;
```
vs my previous version `2.9.3`:
```
private String value;
``` 
I tried to add some `configOptions` for `openapi-generator-maven-plugin` (used for generation java classes) but didn't help.